rust/plugin/src/project/project_driver.rs

```diff
@@ -1283,17 +1280,17 @@ fn clone_doc(repo_handle: &RepoHandle, doc_handle: &DocHandle) -> DocHandle {
 
 fn handle_changes(handle: DocHandle) -> impl futures::Stream<Item = SubscriptionMessage> + Send {
     futures::stream::unfold(handle, |doc_handle| async {
-        let heads_before = doc_handle.with_doc(|d| d.get_heads());
+        let _/*heads_before */ = doc_handle.with_doc(|d| d.get_heads());
         let _ = doc_handle.changed().await;
```

Comment:
Add a TODO note there that this will probably break on upgrading automerge_repo and will need to be refactored, as the `TODO` message on the `changed()` function indicates that the current behavior is not intended